ideascube/monitoring/forms.py

```diff
@@ -73,16 +74,17 @@ class Meta:
 
 class InventorySpecimenForm(forms.ModelForm):
 
-    specimen = forms.CharField(widget=forms.TextInput(attrs={
-                               'placeholder': _('Enter a bar code')}))
+    specimen = forms.CharField(label=_('specimen'),
+                               widget=forms.TextInput(attrs={
+                                'placeholder': _('Enter a barcode')}))
```

Comment:
At first I thought `'placeholder'` had one space too much as indentation, since it wasn't indented with the previous line.

Then I realized it in fact needs t be indented more than `widget=` since it is nested inside.

Indenting with one space is frankly ugly, and just too few to clearly see the needed indentation (the proof being that I missed it at first :wink: )

I'd suggest indenting with four spaces, as usual.